src/widgets/VaccineChart.js

```diff
@@ -155,40 +230,78 @@ export class VaccineChart extends React.Component {
   }
 }
 
+const chartSize = {
+  width: 1,
+  height: 1.025,
+};
+
 const chartStyles = {
   display: 'flex',
   flexDirection: 'row',
   justifyContent: 'center',
+  padding: { top: 20, left: 50, right: 50, bottom: 50 },
```

Comment:
i think this removed the padding, padding should be on the parent elements. I was thinking:
```
const chartStyles = {
 style: { display: 'flex',
 flexDirection: 'row',
 justifyContent: 'center' },
 padding: { top: 20, left: 50, right: 50, bottom: 50 }
}
```

and in
```
 <VictoryChart
   width={width}
   height={height}
    {...chartStyles}
```

Reply:
Ah, I thought I got away with it, it looked the same to me. Will do it this way.